x-pack/docs/en/security/authentication/jwt-realm.asciidoc

```diff
@@ -12,35 +12,61 @@ between the _client_ that is connecting to {es}, and the _user_ on whose behalf
 the request should run. The JWT authenticates the user, and a separate credential
 authenticates the client.
 
-A common scenario for JWTs is when an existing front-end application uses
-OpenID Connect (OIDC) to authenticate and identify a user, and then accesses {es}
-on behalf of the authenticated user.
-
-TIP: If the front-end application does not exist, you can use the
-<<token-authentication-services>> instead.
+The JWT realm is designed to cater for the following two scenarios:
+
+1. An application authenticates and identifies a user with an authentication flow, e.g.
+OpenID Connect (OIDC), and then accesses {es} on behalf of the authenticated user.
+A common scenario for JWTs is when an existing application uses
+2. An application itself goes through an authentication flow, e.g. OAuth2 Client Credentials
+Flow, and then accesses {es} for itself.
+
+In both scenarios, the application should present a JWT to {es} that represents either the user (scenario 1)
+or itself (scenario 2). {es} categorizes the JWT into two types, ID Token and Access token,
+and validates them accordingly. The ID Token type basically follows the OIDC specification.
```

Comment:
I don't love "basically" there. It makes me feel like it's saying we "sort of" follow the spec.
That is true in the sense that what we're doing here isn't covered by the spec, so we've had to adjust as needed, but I don't think that's how readers will interpret it.
I would prefer something like `The ID Token type is designed to authenticate using tokens that follow the OIDC specification.`

Reply:
Good point. I agree "basically" does not read very sincere. I adopted your version.